backend/source/generated/restapi/configure_coreapi.go

```diff
@@ -127,6 +127,7 @@ func setupGlobalMiddleware(handler http.Handler) http.Handler {
 	corsHandler := cors.Handler(cors.Options{
 		AllowedOrigins: []string{"*"},
 		AllowedHeaders: []string{"*"},
+		AllowedMethods: []string{"POST", "GET", "PATCH", "PUT", "OPTIONS", "DELETE", "HEAD"},
```

Comment:
what are the defaults? do we need to allow all of these?

Reply:
Defaults are POST, GET, PUT, DELETE.
We allow them, but we don't have implementation for some of them.